src/Entity/Embargo.php

```diff
@@ -397,29 +397,33 @@ public function getCacheMaxAge() {
    * {@inheritdoc}
    */
   public function getCacheTags() {
-    $tags = Cache::mergeTags(parent::getCacheTags(), $this->getEmbargoedNode()->getCacheTags());
+    if ($this->getEmbargoedNode() !== NULL) {
+      $tags = Cache::mergeTags(parent::getCacheTags(), $this->getEmbargoedNode()->getCacheTags());
 
-    if ($this->getExemptIps()) {
-      $tags = Cache::mergeTags($tags, $this->getExemptIps()->getCacheTags());
+      if ($this->getExemptIps()) {
+        $tags = Cache::mergeTags($tags, $this->getExemptIps()->getCacheTags());
+      }
+      return $tags;
     }
-    return $tags;
   }
```

Comment:
Instead of reindenting ALL the things, just move the one conditional things inside a condition. Also, this should still return an array?

As in, could just deal with with changing from:

```php
$tags = Cache::mergeTags(parent::getCacheTags(), $this->getEmbargoedNode()->getCacheTags());
```

to

```php
$tags = parent::getCacheTags();
if ($node = $this->getEmbargoedNode()) {
  $tags = Cache::mergeTags($tags, $node->getCacheTags());
}
```